webui/react/src/components/InlineEditor.tsx

```diff
@@ -62,14 +62,12 @@ const InlineEditor: React.FC<Props> = ({
 
   const save = useCallback(async (newValue: string) => {
     if (onSave) {
-      try {
-        setIsSaving(true);
-        await onSave(newValue);
-      } catch (e) {
+      setIsSaving(true);
+      const err = await onSave(newValue);
+      if (err !== null) {
```

Comment:
You should be able to just use `if (err)`

Reply:
I get this TypeScript error `An expression of type 'void' cannot be tested for truthiness.  TS1345`
I'm making the other changes but `onSave` has to have a consistent Promise<object> type or keep this if err !== null